extensions/auth/portability-auth-microsoft/src/main/java/org/dataportabilityproject/auth/microsoft/MicrosoftAuthDataGenerator.java

```diff
@@ -30,8 +26,8 @@
       "https://login.microsoftonline.com/common/oauth2/v2.0/token";
 
   private final String redirectPath;
-  private final Supplier<String> clientIdSupplier;
-  private final Supplier<String> clientSecretSupplier;
+  private final String clientIdSupplier;
```

Comment:
nit: can these just be called clientId and clientSecret now?

Reply:
Done